app/scripts/controllers/feed.js

```diff
@@ -127,7 +127,7 @@ const feed = angular.module('feed', ['stats', 'udc', 'utils'])
     // Set default menu data
     $scope.menu = [{
       'url': 'https://crate.io/demo?utm_source=adminui&utm_medium=browser&utm_term=&utm_content=demolink&utm_campaign=newsfeed&ajs_event=clicked_demo_link',
-      'title': 'Schedule a 1-ON-1 demo with a Crate.io engineer'
+      'title': 'Schedule a 1-ON-1 demo with a Crate engineer'
```

Comment:
why that change? at the translation files you wrote `Crate.io ....` (https://github.com/crate/crate-admin/pull/712/files#diff-cfbca3bbdba4f816f75a9bb32f41f9dbc41ce1b20c342b65bab0337e491dbec2R193) Should be consistent at least ;)